src/components/Login/LoginPage.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles, Box, Typography } from '@material-ui/core';
+import LoginButton from './LoginButton';
+import InvalidUserMessage from './InvalidUserMessage';
+import { Colours } from '../../styles/Constants';
+
+import paramedicsLogo from '../../paramedicsLogo.png';
+import paramedicsBackground from '../../paramedicsBackground.png';
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: Colours.White,
+    height: '100vh',
+    display: 'flex',
+  },
+  leftColumn: {
+    paddingTop: '207px',
+  },
+  rightColumn: {
+    textAlign: 'center',
+    paddingTop: '207px',
+    color: Colours.White,
+    backgroundColor: Colours.Secondary,
+    backgroundImage: `url(${paramedicsBackground})`,
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+  },
+  boldText: {
+    fontWeight: 'bold',
+    paddingTop: '40px',
+  },
+  adminAccessText: {
+    width: '329px',
+    paddingBottom: '40px',
+  },
+  loginText: {
+    color: Colours.SecondaryGray,
+    paddingTop: '20px',
+  },
+  appNameText: {
+    paddingTop: '16px',
+    margin: 'auto',
+    width: '366px',
+  },
+  paramedicsText: {
+    paddingTop: '16px',
+    margin: 'auto',
+    width: '220px',
+  },
+  paramedicsLogo: {
+    paddingTop: '16px',
+  },
+});
+
+const LoginPage = ({ invalidUser }: { invalidUser: Boolean }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
+      <Box width="50%" className={classes.leftColumn}>
+        <Box paddingLeft="122px">
+          <Typography variant="h3">Welcome</Typography>
+          <Typography variant="body1" className={classes.boldText}>
+            First time here?
+          </Typography>
+          <Typography variant="body1" className={classes.adminAccessText}>
+            Check that your administrator has granted you access before logging
+            in
```

Comment:
?
```suggestion
            Check that your administrator has granted you access before logging in
```

Reply:
I think this is the linter. Seems like it has no functional impact

Reply:
yup it's the linter, I tried making this one line, but running `npm run lint` forces it to be two